interfaces/ocpp.yaml

```diff
@@ -0,0 +1,111 @@
+  to be used for both OCPP1.6 and OCPP2.0.1 module implementations. Therefore, the vars, commands and types 
+  are based more on the definitions of OCPP2.0.1, as this offers more flexibility and it is easier to transfer
+  to the capabilities of OCPP1.6 than vice versa.
+cmds:
+  stop:
+    description: >-
+      Disconnects the websocket connection and stops the OCPP communication.
+      No OCPP messages will be stored and sent after a restart.
+    result:
+      type: boolean
+      description: >-
+        Returns true if the service could be stopped successfully, else
+        false
+  restart:
+    description: >-
+      Connects the websocket and enables OCPP communication after a previous
+      stop call.
+    result:
+      type: boolean
+      description: >-
+        Returns true if the service could be restarted successfully, else
+        false
+  security_event:
+    description: >-
+      Triggers a SecurityEventNotification.req at the CSMS. This event is queued with a guaranteed delivery
+      to the CSMS.
+    arguments:
+      type:
+        description: type of the security event
+        type: string
+      info:
+        description: Additional information about the occurred security event
+        type: string
+  get_variables:
+    description: >-
+      Command to get a variable from OCPP.
+      With OCPP1.6: Retrieves a configuration key.
+      With OCPP2.0.1: Retrieves a variable with value from the device model storage
+    arguments:
+      requests:
+        description: List of GetVariableRequest
+        type: array
+        items:
+          minimum: 0
+          type: object
+          $ref: /ocpp#/GetVariableRequest
+    result:
+      description: >-
+        List of GetVariableResult containing the result for every requested value. Does not guarantee preservation of order!
+      type: array
+      items:
+        type: object
+        $ref: /ocpp#/GetVariableResult
+  set_variables:
+    description: >-
+      Command to set a variable at OCPP. 
+      With OCPP1.6: This command can be used to set custom configuration keys (others will be rejected)
+      With OCPP2.0.1: This command can be used to set variables in the device model storage
+    arguments:
+      requests:
+        description: List of SetVariableRequests
+        type: array
+        items:
+          minimum: 0
+          type: object
+          $ref: /ocpp#/SetVariableRequest
+    result:
+      description: >-
+        List of SetVariableResult containing the result for every requested set operation
+      type: array
+      items:
+        type: object
+        $ref: /ocpp#/SetVariableResult
+  monitor_variables:
+    description: >-
+      Command to start monitoring the given ComponentVariable(s). In case this configuration keys is 
+      changed by the CSMS the configuration key the changed configuration key will be published. 
```

Comment:
"the configuration key the changed configuration key" -> some words seem to be duplicated. Probably meant: "the changed configuration key"